### PR TITLE
Replacing anchors with buttons

### PR DESCRIPTION
**Pull request by thers**
Would be better for semantic.

### CODE REVIEW

**Comment by thers** [comment]
QUnit was alerting about erros, but i think it's because of staging branch, so don't mind about it :)

File: lib/alertify.js [hunk 1]
```diff
@@ -33,9 +33,9 @@
 		dialogs = {
 			buttons : {
 				holder : "<nav class=\"alertify-buttons\">{{buttons}}</nav>",
-				submit : "<button type=\"submit\" class=\"alertify-button alertify-button-ok\" id=\"alertify-ok\">{{ok}}</button>",
-				ok     : "<a href=\"#\" class=\"alertify-button alertify-button-ok\" id=\"alertify-ok\">{{ok}}</a>",
-				cancel : "<a href=\"#\" class=\"alertify-button alertify-button-cancel\" id=\"alertify-cancel\">{{cancel}}</a>"
+				submit : "<button type=\"submit\" role="button" class=\"alertify-button alertify-button-ok\" id=\"alertify-ok\">{{ok}}</button>",
+				ok     : "<button type=\"button\" role="button" class=\"alertify-button alertify-button-ok\" id=\"alertify-ok\">{{ok}}</button>",
+				cancel : "<button type=\"button\" role="button" class=\"alertify-button alertify-button-cancel\" id=\"alertify-cancel\">{{cancel}}</button>"
```
Comment (fabien-d) [inline lib/alertify.js:38]: forgot to escape the quotes around the `role` attribute `"` should be `\"`.

**Comment by fabien-d** [comment]
The tests don't run because of linting errors. If you fix the quotes issue the tests should pass.

**Comment by thers** [comment]
Uh, im so sorry :( first contributing to open source, exciting %) I think at now it would be better if you change this by yourself instead of me. I apologize for disturbing.

**Comment by fabien-d** [comment]
I released `0.4.0rc1` and I've included this as part of it. If you want to submit another PR so it's part of the `0.3.*` release, fix the escaping and submit to the `0.3` branch.

**Pull request closed without merging**